### PR TITLE
KOGITO-3786: Jobs management pagination

### PR DESCRIPTION
**Pull request by Sara4994**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: ui-packages/packages/common/src/graphql/graphql.schema.json [hunk 1]
```diff
@@ -153,6 +153,147 @@
             },
             "isDeprecated": false,
             "deprecationReason": null
+          },
+          {
+            "name": "Travels",
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/graphql/graphql.schema.json]: Why are the domain-specific terms like Travels, Visa, etc mentioned in common package? The query should be generalized imo and the actual domain passed as argument.

File: ui-packages/packages/common/src/graphql/graphql.schema.json [hunk 2]
```diff
@@ -4768,185 +4513,2586 @@
...
+        "inputFields": null,
+        "interfaces": [],
         "enumValues": null,
         "possibleTypes": null
       },
       {
-        "kind": "INPUT_OBJECT",
-        "name": "KogitoMetadataOrderBy",
+        "kind": "OBJECT",
+        "name": "Flight",
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/graphql/graphql.schema.json]: again, should not be here.

File: ui-packages/packages/common/src/graphql/types.tsx [hunk 3]
```diff
@@ -366,6 +486,95 @@ export namespace GraphQL {
     UserTaskInstanceUpdated: UserTaskInstance;
     JobAdded: Job;
     JobUpdated: Job;
+    TravelsAdded: Travels;
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/graphql/types.tsx]: Should not be handled like this - Travels, Visa, Deals are domain-specific, whereas Job and UserTask are general entities. I think this is not correct approach to take.

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
I think the general approach should be different.

**Comment by Sara4994** [comment]
@jstastny-cz yep that's my bad, corrected both the files. Thanks

**Comment by jstastny-cz** [comment]
@Sara4994 seems to work fine in dev mode, but there's one snapshot test failure, please check.

**Comment by Sara4994** [comment]
jenkins retest this please

**Review by AjayJagan — APPROVED** [review]
:+1:

**Comment by Sara4994** [comment]
jenkins retest this please

**Review by jstastny-cz — APPROVED** [review]

**Review by nmirasch — APPROVED** [review]

**Pull request merged**